config.ru

```diff
@@ -0,0 +1,8 @@
+#Setup Load Path
+$LOAD_PATH << File.expand_path("../lib", __FILE__)
+
+require './lib/tic_tac_toe_web_app'
```

Comment:
Since you modified the load path right above, I think this should read `require 'tic_tac_toe_web_app'`. Avoid putting unneeded paths in your requires otherwise you won't be able to move files around.